HW3/task_1.py

```diff
@@ -0,0 +1,22 @@
+zen_of_python = """
+Beautiful is better than ugly. Explicit is better than implicit. Simple is better than complex.
+Complex is better than complicated. Flat is better than nested. Sparse is better than dense.
+Readability counts. Special cases aren't special enough to break the rules. Although practicality
+beats purity. Errors should never pass silently. Unless explicitly silenced. In the face of
+ambiguity, refuse the temptation to guess. There should be one-- and preferably only one --obvious
+way to do it. Although that way may not be obvious at first unless you're Dutch. Now is better than
+never. Although never is often better than *right* now. If the implementation is hard to explain,
+it's a bad idea. If the implementation is easy to explain, it may be a good idea. Namespaces are
+one honking great idea -- let's do more of those!
+"""
+#find separately the number of occurrences of the words "better", "never" and "is" in the given line
+print(zen_of_python.find('better'))
```

Comment:
in this case you need to use method count() 
look at this code:
print(zen_of_python.count('better'))
print(zen_of_python.count('never'))
print(zen_of_python.count('is'))